src/sorts.tsx

```diff
@@ -64,6 +64,18 @@ const SORTS: Sort[] = [
     name: "Callsign",
     impl: (a, b) => a.callsign.localeCompare(b.callsign),
   },
+  {
+    name: "Type",
+    impl: (a, b) => a.cardType.localeCompare(b.cardType),
```

Comment:
might want to sort each section by something else (e.g. how starred also sorts by last name)

Reply:
Was thinking about this too, probably last name/location